chessboard/script.js

```diff
@@ -0,0 +1,97 @@
+const board = document.querySelector(".board");
```

Comment:
Hi @shivam907, Your code is logically correct and you have taken care of device responsiveness as well which is a plus point, +1 for that